packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -32,12 +32,24 @@ options={{
 
 export const Docs: () => JSX.Element = () => {
   const refTeamsMeeting = useRef(null);
+  const refFileSharing = useRef(null);
+  const refFileSharingACS = useRef(null);
+  const refFileSharingTeams = useRef(null);
+
+  const scrollToRef = (ref): void => {
+    ref.current.scrollIntoView({ behavior: 'auto' });
+  };
 
   useEffect(() => {
     const url = document.URL;
-
-    if (url.includes('teams-meeting') && refTeamsMeeting.current) {
-      refTeamsMeeting.current.scrollIntoView({ behavior: 'auto' });
+    if (url.includes('joining-a-teams-meeting') && refTeamsMeeting.current) {
```

Comment:
why is `joining-a-teams-meeting` rather than `join-teams-meeting`?

Reply:
In the URL provided for the storybook— "https://60c7ae6891f0e90039d7cd54-ppikpqkivw.chromatic.com/?path=/docs/composites-call-with-chat-jointeamsmeeting--join-teams-meeting#in-teams-interop-meeting-chat-thread"
—you'll notice that it contains both `join-teams-meeting` and `in-teams-interop-meeting-chat-thread`. If you set up your first condition as `if (url.includes('joining-a-teams-meeting') && refTeamsMeeting.current) {`, the code will execute `scrollToRef(refTeamsMeeting);`. This will scroll to the "Teams Meeting" section instead of the intended "in-teams-interop-meeting-chat-thread" section. Is that the desired behavior?

Reply:
Oh, never mind. `joining-a-teams-meeting` should also work

Reply:
the first if is for existing navigation to this page: 
<img width="1164" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/bb65b806-df47-4036-952f-264940a2983c">
why would the first if gets triggered when the term "in-teams-interop-meeting-chat-thread" does not contain "joining-a-teams-meeting"?

Reply:
However, I'd like to note that using `joining-a-teams-meeting` in the condition would also restrict scrolling to subsections under "Join a Teams Meeting." For example, if the URL is "https://60c7ae6891f0e90039d7cd54-ppikpqkivw.chromatic.com/?path=/docs/composites-call-with-chat-jointeamsmeeting--join-teams-meeting#customize-call-controls," the page will not scroll to the "Join a Teams Meeting" section as one might expect.